indexer/database/db.go

```diff
@@ -91,3 +97,14 @@ func dbFromGormTx(tx *gorm.DB) *DB {
 		BridgeTransactions: newBridgeTransactionsDB(tx),
 	}
 }
+
+func (db *DB) executeSQLMigration() error {
```

Comment:
Lest pull this codeblock from the e2e test setup https://github.com/ethereum-optimism/optimism/blob/develop/indexer/e2e_tests/setup.go#L141 into this.

We should walk the migrations directory since after launch any changes should be altering existing schemas in a new file

Reply:
And with this public, we can call it in the test setup as well

Reply:
Done!